pkg/utils/tekton/pipelines.go

```diff
@@ -87,6 +89,23 @@ type VerifyEnterpriseContract struct {
 }
 
 func (p VerifyEnterpriseContract) Generate() *v1beta1.PipelineRun {
+	var snapshot app.SnapshotSpec
+	err := json.Unmarshal([]byte(p.Image), &snapshot)
```

Comment:
Feels wrong to parse the Image into a `app.SnapshotSpec`, why not have the `Snapshot app.SnapshotSpec` as a field in `VerifyEnterpriseContract` struct? Would be good opportunity to refactor `VerifyEnterpriseContract` and replace `ApplicationName`, `ComponentName`, `Image` with `Snapshot`

Reply:
Thanks @zregvart for the propose .
The reason I choose parse Image into `app.SnapshotSpec` is want to keep the most of `ec validate image` command.
For image input , simplely input "image url", and the logic in this pr will compose a json-input image for task "verify-enterprise-contract". If I have `Snapshot app.SnapshotSpec` in `VerifyEnterpriseContract `struct, before "image url" takes as input, it should be compose to a snapshot. For other team who want to intergrate with ec, they need to ensure they put  a `Snapshot app.SnapshotSpec` not only a "image url".

Reply:
create [HACBS-2200](https://issues.redhat.com/browse/HACBS-2200) for a following  PR to address this comments.